### DISCUSSION TITLE
Give Deadeye Carbine burst fire

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#6201] [Ideas and suggestions]
Suggesting:
4 round burst for Deadeye Carbine over 0.2 seconds, with a reload time of 1 second.

Giving Deadeye Carbine burst fire capability would bring it up to, well, not necessarily par with its baseline SMG, but no longer usually worse.

Is it worse usually, right now? Well, at the effective ranges of most encounters with locals, the carbine's advantage is negligible (SMG accurate enough to hit), and its ROF compared to its stun duration is disappointing at best, and much worse than SMG. 

This will also make it feel really cool.

**Comment by Tsunder** [comment]
https://user-images.githubusercontent.com/5103517/125212816-dfe44300-e27d-11eb-8a2a-c339eea3fade.mp4

**Comment by Reed0506** [comment]
very cool, rewards the player for skill rather than punishing them for a largely inconsequential increase to statistical accuracy.

**Comment by Tsunder** [comment]
was implemented in #6577
